README.md

```diff
@@ -0,0 +1,28 @@
+# Juju GUI
+
+The Juju GUI is a web-based GUI for [Juju](https://jujucharms.com). Juju allows
+you to deploy, configure, manage, maintain, and scale cloud applications quickly
+and efficiently on public clouds, as well as on physical servers, OpenStack, and containers.
+
+The Juju GUI is open source and the code is available on [GitHub](https://github.com/juju/juju-gui).
+
+### Accessing the GUI
+
+The latest release of the Juju GUI is made available to all users of Juju automatically
+and can be launched by running `juju gui` in your terminal. For those using
+[JAAS](https://jujucharms.com) you can use the GUI by logging into your account
```

Comment:
For those using JAAS, ... (add the comma).

Reply:
added